applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -288,7 +288,9 @@ int ElasticIsotropic3D::Check(
     KRATOS_CHECK_VARIABLE_KEY(POISSON_RATIO);
     const double& nu = rMaterialProperties[POISSON_RATIO];
     const bool check = static_cast<bool>((nu >0.499 && nu<0.501) || (nu < -0.999 && nu > -1.01));
-    KRATOS_ERROR_IF(check) << "POISSON_RATIO is invalid value " << std::endl;
+    if (check) {
```

Comment:
0.499 is a valid value!!

for example in our tests we use 0.4995!, and in incompressible literature it is typical to use more 9s

Reply:
I know, but this is like it is right now, that's why I chnaged for a wraning, in any case that will be a future PR